src/test/java/com/rocketden/main/socket/RoomSocketTests.java

```diff
@@ -98,24 +99,64 @@ public void handleFrame(StompHeaders headers, Object payload) {
                 blockingQueue.add((RoomDto) payload);
             }
         });
+    }
+
+    @Test
+    public void socketReceivesMessageOnJoin() throws Exception {
```

Comment:
Thanks for adding this! This is unrelated to your current PR, right? This is a great test to add either way.